src/index.html

```diff
@@ -22,6 +22,327 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-tag"
+        href="#"
+      >
+        <img
+          class="header__logo-image"
+          src="./images/logo.png"
+          alt="MOYO_LOGO"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card">
```

Comment:
"card-list" or "card-grid" or just "grid" would all be better names here, because this doesn't refer to a single card, but the whole group of it